src/util/http-date.js

```diff
@@ -0,0 +1,43 @@
+const dayNames = [
+  'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'
+];
+const monthNames = [
+  'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+  'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+];
+
+/**
+ * Formats a `Date` object according to RFC 7231 (used for Last-Modified header)
+ *
+ * @param {Date} dt   - The Date to format
+ * @returns {String}  - The formatted string
+ */
+function formatDate(dt) {
+  const day = dayNames[dt.getUTCDay()];
+  const month = monthNames[dt.getUTCMonth()];
+  const date = `${day}, ${dt.getUTCDate()} ${month} ${dt.getUTCFullYear()}`;
+  const time = [dt.getUTCHours(), dt.getUTCMinutes(), dt.getUTCSeconds()]
+    .map(zeroPad2)
+    .join(':');
+  return `${date} ${time} GMT`;
```

Comment:
Why not just use `dt.toUTCString()` ??

Reply:
Because that's not the correct date format for `Last-Modified`.

Reply:
You're right 🤦‍♂. Will swap that in.

Reply:
Done. Tests still pass with built-in date formatting.